app/view/info/appsView.js

```diff
@@ -172,6 +172,13 @@ SDL.InfoAppsView = Em.ContainerView.create({
     /** Items */
     items: new Array()
   }
-)
+),
+
+  imageModeChanged: function() { 

```

Comment:
@YarikMamykin please add a description for all observed functions and what are they doing

Reply:
@AKalinich-Luxoft description added in https://github.com/LuxoftSDL/sdl_hmi/pull/10/commits/2552a02d0bc671b9b4458954f5a17012f207148d https://github.com/LuxoftSDL/sdl_hmi/pull/10/commits/ff255ffa94d195ab2a718312b6fb505ad83c901c https://github.com/LuxoftSDL/sdl_hmi/pull/10/commits/c31f95aab7a862cbea07b86d3ccb05cb700506a8 https://github.com/LuxoftSDL/sdl_hmi/pull/10/commits/eb0f480176535219fc1c59080bf986efd69bef30 https://github.com/LuxoftSDL/sdl_hmi/pull/10/commits/352785d8e34f358e0705ff618424f52dd037a4ab